lib/nerves_hub_link/socket.ex

```diff
@@ -270,13 +273,39 @@ defmodule NervesHubLink.Socket do
     {:noreply, disconnect(socket)}
   end
 
-  def handle_cast({:send_update_progress, progress}, socket) do
-    _ = push(socket, @device_topic, "fwup_progress", %{value: progress})
+  def handle_cast({:send_firmware_update_progress, :progress, progress}, socket) do
+    _ =
+      push(socket, @device_topic, "firmware_update_status", %{
+        status: "progress",
+        percent: progress
+      })
+
+    {:noreply, socket}
+  end
+
+  def handle_cast({:send_firmware_update_status, :finalizing, payload}, socket) do
+    {:ok, push_ref} =
+      push(
+        socket,
+        @device_topic,
+        "firmware_update_status",
```

Comment:
This new event name couples itself to a NervesHub update. May not a bad thing, but just calling out for history

Reply:
Sorry, can you go a bit deeper into this, I'm not sure what the downside is you are referring to.